README.md

```diff
@@ -14,78 +16,78 @@ nRF Connect supports the following devices:
 * PCA10059 nRF52840 Dongle
 * PCA10090 nRF9160 Development Kit
 
-Note that while the nRF Connect framework has support for all these devices, the device support in nRF Connect apps may vary.
-
-# Installation
-
-To install nRF Connect you can download binaries from the [nRF Connect product page](https://www.nordicsemi.com/eng/Products/Bluetooth-low-energy/nRF-Connect-for-Desktop#Downloads) on Nordic Semiconductor web pages.
+Note that while the the framework has support for all these devices, the device support in individual apps may vary.
 
-nRF Connect currently supports the following operating systems:
+You can run nRF Connect for Desktop on:
 
 * Windows
 * Ubuntu Linux 64-bit
 * macOS
 
-# Usage documentation
+# Using nRF Connect for Desktop
+
+## Prerequisites
+
+### macOS
 
-A [Getting started guide](https://www.nordicsemi.com/eng/Products/Bluetooth-low-energy/nRF-Connect-for-Desktop#Documentation) is available from the nRF Connect product pages.
+J-Link driver needs to be separately installed on macOS. Download and install it from [SEGGER](https://www.segger.com/downloads/jlink) under the section *J-Link Software and Documentation Pack*. Without it when running the apps you would otherwise get error messages when running the apps like `CouldNotOpenDLL` or `JLINKARM_DLL_NOT_FOUND`.
+
+### Linux
+
+J-Link driver needs to be separately installed on Linux. Download and install it from [SEGGER](https://www.segger.com/downloads/jlink) under the section *J-Link Software and Documentation Pack*. Without it when running the apps you would otherwise get error messages when running the apps like `CouldNotFindJprogDLL`.
```

Comment:
Maybe Linux and macOS prereqs could be stated in one paragraph to avoid duplication. Then the udev sentence could start like: "On Linux ..."

Reply:
Yes, merged them again in 5f4ff5e42fa9d82b12db0eabca6815bdc9d27896.